### PR TITLE
Helmfile

### PR DESCRIPTION
**Pull request by ben851**
Basic helmfile deployment to bdm dev working.

### CODE REVIEW

File: bdmDevContext.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+az account set -s EsDBDMDECDSub
+export SUBSCRIPTION_ID=$(az account show --subscription "EsDBDMDECDSub" --query 'id' -o tsv)
+export TENANT_ID=$(az account show --subscription "EsDBDMDECDSub" --query 'homeTenantId' -o tsv)
+export KEYVAULT_NAME=esdcdpsbdmk8sdev-KV
+export KEYVAULT_READ_USER=$(az keyvault secret show --vault-name $KEYVAULT_NAME --name k8s-keyvault-sp-id --query value -otsv)
+export KEYVAULT_READ_PASSWORD=$(az keyvault secret show --vault-name $KEYVAULT_NAME --name k8s-keyvault-sp-pass --query value -otsv)
+export K8S_CLUSTER_NAME=ESdCDPSBDMK8SDev-K8S
+export K8S_RG_NAME=ESdCDPSBDMK8SDev
+export BASE_DOMAIN=bdm-dev.dts-stn.com
+az aks get-credentials --name $K8S_CLUSTER_NAME --resource-group $K8S_RG_NAME
```
Comment (JeremyKennedy) [inline bdmDevContext.sh]: Is this file necessary in the repo, or was it just a temporary solution?

File: helmfile/helmfile.yaml [hunk 2]
```diff
@@ -14,13 +14,13 @@ environments:
 
 releases:
 # Single Tier Website
-  - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
-    chart: ../helm/Charts/single-tier-appliation
+  - name: {{ requiredEnv "PROJECT" }}-{{ .Environment.Name }}
+    chart: ../helm/charts/single-tier-application
     values:
       - overrides/eligibility-estimator-client/eligibility-estimator-client.yaml.gotmpl
```
Comment (JeremyKennedy) [inline helmfile/helmfile.yaml]: would it be okay if I renamed these paths (and also updated any references elsewhere), or should I stick with it as is? I'd like to use `coabe` instead of `eligibility-estimator-client` if possible. I can make the change and push it here if that's cool!

**Review by JeremyKennedy — COMMENTED** [review]
Couple tiny things I would like to clean up if possible, but if not, it is great as is! Thank you for the support today in getting this working.

File: helmfile/helmfile.yaml (hunk 2 above)
Comment (ben851) [reply helmfile/helmfile.yaml]: This is 100% up to you!

File: bdmDevContext.sh (hunk 1 above)
Comment (ben851) [reply bdmDevContext.sh]: It can be removed.. feel free to do so or I can do it tomorrow.

File: helmfile/helmfile.yaml (hunk 2 above)
Comment (JeremyKennedy) [reply helmfile/helmfile.yaml]: I have pushed a commit renaming references from `eligibility-estimator-client` to `coabe`. I applied this in some places I'm not sure if I should have, specifically within `helm/charts/single-tier-application/Chart.yaml` and `values.yaml`. I decided to leave `single-tier-application` as-is rather than rename to `coabe` for now as I wasn't sure if that made sense to do or not. I figured "single tier application" represented the type of app it was, rather than the name of the app itself, but I guess it doesn't matter much anyways.

Either way, I hope this still works, let me know if I'm okay to merge it in or not, and if there's anything else I should change. Note that I plan on pulling in the changes from the template in a separate PR - I wanted to keep this PR's scope limited.

Comment (JeremyKennedy) [reply helmfile/helmfile.yaml]: Hoping that these changes are okay, and proceeding with the merge.

**Review by JeremyKennedy — APPROVED** [review]
I'm hoping that my changes are okay, and am proceeding with the merge.
